x11-packages/e16/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE=https://www.enlightenment.org/e16
+TERMUX_PKG_DESCRIPTION='E16 (Enlightenment) is a lightweight, highly customizable window manager for X11, known for its unique aesthetic, efficiency, and low resource usage, making it ideal for older or limited-resource systems.'
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.30
+TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/enlightenment/e16-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=b8b7748a2c48c4c7c9758d9ad12b14f566d2bec38f2eda533e6d874f5ce9074c
+TERMUX_PKG_DEPENDS="imlib2"
```

Comment:
```suggestion
TERMUX_PKG_DEPENDS="glib, imlib2, libiconv, libx11, libxext, libxft, libxrender, pango"
```

Reply:
Usually you can use project documentation and other build scripts for the same package (https://repology.org/projects/) as a guide.
`ldd` can also be helpful, though that only shows what the executable or shared object links to,
so you'd need to use it on a compiled version of the package.

Reply:
I crafted a bash function that uses readelf and apt-file for identifying dependencies on device to review PRs.
I am waiting #21847 or looking into whether its possible to do it on CI...